okapi-core/src/main/raml/okapi.raml

```diff
@@ -706,6 +715,15 @@ types:
             disable current module with new module ID in body and existing ID
             in path.
             This call will eventually be replaced by the 'install' service.
+          queryParameters:
+            purge:
+              description: Disabled modules will also be purged.
+              type: boolean
+              required: false
+            tenantParameters:
+              description: Parameters for Tenant init
+              type: string
+              required: false
           body:
```

Comment:
Does this section (mdTo will not be empty, right?) conflict with line 400 `boolean purge = mdTo == null && options.getPurge();`

Reply:
purge would always be false in this case. mdTo == null ONLY if a module is disabled (no downgrade or upgrade). The purge would be ignored essentially.

Reply:
So this part should be modified because essentially this API does not support "purge" parameter, right?